src/SmartComponents/CompletedTaskDetails/Columns.js

```diff
@@ -15,7 +15,7 @@ const SystemNameCell = ({ /*id,*/ display_name }, index) => (
 
 SystemNameCell.propTypes = {
   id: propTypes.string,
-  display_name: propTypes.string,
+  display_name: propTypes.any,
```

Comment:
FYI there is `propTypes.node` which means "anything that can be rendered" and might be a bit more appropriate here